cranelift/codegen/src/isa/x64/inst/emit_tests.rs

```diff
@@ -2408,21 +2408,6 @@ fn test_x64_emit() {
         "movsd   %xmm14, %xmm3",
     ));
 
-    // ========================================================
-    // XMM_R_R
-
-    insns.push((
-        Inst::xmm_r_r(SseOpcode::Movss, xmm3, w_xmm2),
```

Comment:
I would have assumed that the tests could be kept, but using the `xmm_mov_rm_r` constructor instead of the `xmm_r_r`, and moving these close to the other `xmm_mov_rm_r` tests. Can you do this here please? (Unless this is a redundant ask because similar tests existed with the other Inst)

Reply:
Hi .. yes. I included redundant tests earlier when creating the xmm_mov_rm_r structure: See here: https://github.com/bytecodealliance/wasmtime/blob/f2f770626511c08c632c52e5f868ac13170687be/cranelift/codegen/src/isa/x64/inst/emit_tests.rs#L2400-L2409